rpc/src/v1/traits/chain.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2019 Kodebox, Inc.
+// Copyright 2018-2020 Kodebox, Inc.
```

Comment:
Please update the copyrights in the files that you changed.

Reply:
Sorry. I changed the copyright of wrong file. I pushed the fixed version.